src/tox/session/commands/show_config.py

```diff
@@ -29,7 +29,10 @@ def show_config(config):
     content = StringIO()
     parser.write(content)
     value = content.getvalue().rstrip()
-    reporter.verbosity0(value)
+    # We are not using the reporter because showconfig result should not be
+    # affected by logging level as its stdout output *is* expected to
+    # be machine parseable.
+    print(value)
```

Comment:
> So this PR is about making it behave the same as -l and -a

This is not true. https://github.com/tox-dev/tox/blob/master/src/tox/session/commands/show_env.py#L22 is not using print.

Reply:
What is not true? The behaviour will not be consistent, after the PR is merged? Ok then, how can we make it consistent? 

You asked for the opinion of other maintainers. I was out of the loop for a while and I am trying to help to solve a dispute without having any skin in the game myself. I reviewed the changes and gave my opinion and it is: I think it would be good if the 3 machine readable commands would be behave consistently and with my limited understanding that I have about the code base in its current state, the PR solves that problem. If it doesn't do it in the correct way according to your (undoubtably better) understanding, then you are the one who is in the best position to know how to do advise how to do it.

So, my question is: if this is **not** the correct way to make the behaviour of the three commands consistent (and I think we agree that this would be a GoodThing(TM)), then what **is** the correct way?

I also still don't see the danger in these changes and it would help if you could explain for the less experienced developers in the project, what the danger exactly is that you perceive.

Reply:
> What is not true?

Linked you the line showing that's not true. What's happening on master is not what's being changed to.

> , the PR solves that problem

It does so by changing the behaviour of all three commands, not by bringing show config in line with the other two. Also, the new commands side-steps our timestamp printing logic built-in into the reporter. These commands are only meant to be machine-readable when the verbosity is <0. The change here breaks that by trying to make it always machine-readable.

The danger is breaking workflows that actually use the current behaviour.